concordium-sdk/src/main/java/com/concordium/sdk/transactions/TransactionType.java

```diff
@@ -40,4 +40,13 @@ public enum TransactionType {
     TransactionType(byte type) {
         this.value = type;
     }
+
+    public static TransactionType parse(byte type) {
+        for (TransactionType transactionType : values()) {
+            if (transactionType.value == type) {
+                return transactionType;
+            }
+        }
+        throw new IllegalArgumentException("Unknown transaction type tag");
```

Comment:
Probably useful to add the `byte` to this exception for easier debugging.

Reply:
Good idea.